Jenkinsfile

```
@@ -1 +1,12 @@
-buildPlugin()
\ No newline at end of file
+/*
+ See the documentation for more options:
+ https://github.com/jenkins-infra/pipeline-library/
+*/
+buildPlugin(
+  forkCount: '1C', // Run a JVM per core in tests
+  // we use Docker for containerized tests
```

The comment does not match the value of the parameter.  If the plugin needs to use containers for tests, then it needs to `useContainerAgent: false` .  We generally can't use Docker containers for tests when running inside a container agent.
```suggestion
  // we use Docker for containerized tests
```